### PR TITLE
UI Adjustments + Other things

### PR DESCRIPTION
**Pull request by Raffox97**
Italian translation:
- Added a missing country
- Fixed wrong capitalization in some phrases

UI Adjustments:
- Moved colorblind mode in a more visible position (and made it larger)
- Made the "2" in the "CO2" a subscript
- Edited a string to make it translatable

### CODE REVIEW

File: web/views/pages/index.ejs [hunk 1]
```diff
@@ -159,7 +163,7 @@
                                     <div id="country-emission-rect" class="country-col-box emission-rect emission-rect-overview">
                                         <div><span class="country-emission-intensity"></span>g</div>
                                     </div>
-                                    <div class="country-col-subtext">(CO2eq/kWh)</div>
+                                    <div class="country-col-subtext">(CO<sub>2</sub>eq/kWh)</div>
```
Comment (corradio) [inline web/views/pages/index.ejs]: CO2 appears a lot of places (including in specific translations.
Either we replace all CO2 by CO<sub>2</sub>, or we leave it like this.
I think it's better to leave it like it was before to be honest.

File: web/views/pages/index.ejs [hunk 2]
```diff
@@ -310,7 +314,7 @@
                 <small>(<span id="capacity-factor-detail"></span>)</small><br />
                 <br />
                 <%= __('tooltips.withcarbonintensity') %><br />
-                <div class="emission-rect"></div> <span class="emission-intensity"></span> gCO2eq/kWh <small>(source: <span class="emission-source"></span>)</small>
+                <div class="emission-rect"></div> <span class="emission-intensity"></span> gCO<sub>2</sub>eq/kWh <small>(<%= __('country-panel.source') %>: <span class="emission-source"></span>)</small>
```
Comment (corradio) [inline web/views/pages/index.ejs]: Nice catch!

File: web/views/pages/index.ejs [hunk 3]
```diff
@@ -116,10 +116,14 @@
                         <div class="layer-toggles">
                             <hr />
                             <p>
-                                <%= __('legends.carbonintensity')%> <span style="color: darkgrey">(gCO2eq/kWh) </span><small><input type="checkbox" id="checkbox-colorblind" /><label for="checkbox-colorblind"><%= __('legends.colorblindmode') %></label></small>
+                                <%= __('legends.carbonintensity')%> <span style="color: darkgrey">(gCO<sub>2</sub>eq/kWh):</span>
                                 <svg class="co2-colorbar colorbar" style="width: 100%"></svg>
                             </p>
                             <p>
+                                <input type="checkbox" id="checkbox-colorblind" />
```
Comment (corradio) [inline web/views/pages/index.ejs]: I'm not a fan of giving so much visibility to the color blind mode.
I think it should be somehow integrated to the colorbar, and not have its own line.
Depending on what you disliked in the previous version, we could discuss potential changes.
In the meantime, can you revert behavior?

**Comment by Raffox97** [comment]
Should be fixed now

**Comment by Raffox97** [comment]
Sorry, didn't see all that before.

**Comment by corradio** [comment]
No worries - however there's still the color blind checkbox that hasn't been reverted (unless you really have something against the previous version, we can discuss that!)

**Comment by Raffox97** [comment]
Well, I moved down the colorblind option down the wind and the solar one, because I think that that two options are more important than the colorblind, however, there's two reasons why I didn't reverted the position of the checkbox, the first is that some translation (Just like in Italian) are too long to stay in one line (for now), and the second reason is that I think that that option should be more visible.

**Comment by corradio** [comment]
OK sounds good to me. I might move it a bit but I'll try to keep the visibility and sentence length.

**Comment by Raffox97** [comment]
Perfect

**Pull request merged**